tests/test-http-client-transport/src/test/java/org/eclipse/jetty/http/client/HttpClientContinueTest.java

```diff
@@ -319,12 +319,10 @@ public void onComplete(Result result)
 
     @ParameterizedTest
     @ArgumentsSource(TransportProvider.class)
-    @Tag("Slow")
-    @DisabledIfSystemProperty(named = "env", matches = "ci") // TODO: SLOW, needs review
     public void testExpect100ContinueWithContentWithResponseFailureBefore100Continue(Transport transport) throws Exception
     {
         init(transport);
-        long idleTimeout = 1000;
+        long idleTimeout = 100;
```

Comment:
I'm not sure I like this, as 100 ms may be a too short time for the request to even arrive to the server due to class loading or GC on the client.

Reply:
* use AtomicReference to hold `Request`.
* send request
* on server get client-side request and abort it instead of using client-side idle timeout as a mean to cause a failure.
* on server, wait for the request to be completed on the client, then exit `handle()`